dts/bindings/clock/nxp,mcxc-osc.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP MCXC oscillator
+
+compatible: "nxp,mcxc-osc"
+
+include: fixed-clock.yaml
+
+properties:
+  load-capacitance-femtofarads:
+    type: int
+    enum:
+      - 0
+      - 2000
+      - 4000
+      - 6000
+      - 8000
+      - 10000
+      - 12000
+      - 14000
+      - 16000
+      - 18000
+      - 20000
+      - 22000
+      - 24000
+      - 26000
+      - 28000
+      - 30000
```

Comment:
why are you using femto units when all values seem to be scaled by 1000?

Reply:
There is no apparent reason. I will change it to picofarads.

Reply:
Changed to picofarads. @gmarull could you review, please?